.travis.yml

```diff
@@ -1,10 +0,0 @@
-language: csharp
```

Comment:
Why did you delete this file? The CI can't run without it.

Reply:
Will not touch this file in new PR